### ISSUE TITLE
Specification is not clear about how to use libraries

### ISSUE THREAD

**Issue by woodp** [raml-org/raml-spec#606]
When applying libraries some people get confused and think they have to use the include keyword.

See these questions on stack overflow:

http://stackoverflow.com/questions/42427263/mulesoft-error-ramlparser-loadpath-loadapi-content-object-object-no-such/42439035

http://stackoverflow.com/questions/42397718/raml-include-strange-behavior/42398663

Although examples are correct I could not find anywhere in the [specification doc](https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md/) that states clearly that the "include" keyword should not be used when applying libraries ("uses:")

**Comment by sichvoge** [comment]
I completely agree with you. I would probably add a sentence to the third paragraph inside the [Applying Libraries](https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md#applying-libraries) chapter. Something like:

**OLD:**

> Any number of libraries can be applied by using the OPTIONAL uses node ONLY at the root of a ["master"] RAML or RAML fragment file. The value of the uses node is a map of key-value pairs. The keys are treated as library names or namespaces, and the value MUST be the location of a RAML library file, usually an external RAML library fragment document.

**NEW:**

> Any number of libraries can be applied by using the OPTIONAL uses node ONLY at the root of a ["master"] RAML or RAML fragment file. The value of the uses node is a map of key-value pairs. The keys are treated as library names or namespaces, and the value MUST be the location of a RAML library file, usually an external RAML library fragment document. Other than including a fragment file, the location is defined without using the `!include` tag.
